apps/src/templates/studioHomepages/RecentCourses.jsx

```diff
@@ -69,7 +69,7 @@ const RecentCourses = React.createClass({
               isRtl={isRtl}
             />
           )}
-          {!isTeacher && (
+          {!isTeacher && (courses.length > 0 || studentTopCourse) && (
```

Comment:
This is just the inverse of the logic on line 82, right? Should we just create a variable `showSetupMessage = courses.length === 0 && !studentTopCourse` and then use that on line 82, and here do `!isTeacher && !showSetupMessage`?